texlive.profile

```diff
@@ -0,0 +1,30 @@
+selected_scheme scheme-basic
+TEXDIR /app/extensions/TexLive/2018
+TEXMFCONFIG ~/.texlive2018/texmf-config
```

Comment:
I don't know when these are used but apps should be entirely contained to its private XDG dirs.